custom_components/wyzeapi/__init__.py

```diff
@@ -112,19 +163,21 @@ def get_uid():
         mac_addresses.add(hms_id)
 
     device_registry = await dr.async_get_registry(hass)
-    for device in dr.async_entries_for_config_entry(device_registry, config_entry.entry_id):
+    for device in dr.async_entries_for_config_entry(
+        device_registry, config_entry.entry_id
+    ):
         for identifier in device.identifiers:
             domain, mac = identifier
             if mac not in mac_addresses:
-                _LOGGER.warning(f"{mac} is not in the mac_addresses list. Removing the entry...")
+                _LOGGER.warning(
+                    f"{mac} is not in the mac_addresses list. Removing the entry..."
+                )
                 device_registry.async_remove_device(device.id)
     return True
 
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
-    client: Wyzeapy = hass.data[DOMAIN][entry.entry_id][CONF_CLIENT]
-    await client.async_close()
```

Comment:
Why is this being removed? It might not clean up the aiohttp session and client.

Reply:
In the other commit I have for Wyzeapy, I cleaned up the aiohttp requests so that they auto close. I got tired of seeing aiohttp cry about them being left open. So, they'll never be in a state where they get left hanging.